packages/content-conformance/src/rules/top-of-file-h1.js

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line import/no-anonymous-default-export
+/** @type {import('../types.js').ConformanceRuleBase} */
+export default {
+  type: 'content',
+  id: 'top-of-file-h1',
+  description: 'Each content file must contain an H1 the top.',
+  executor: {
+    async contentFile(file, context) {
+      file.visit(['root'], (node) => {
+        const firstNode = node.children[0]
+        const reportMessage = `- ${file.path} \nThe file listed above is missing a level 1 heading at the top of the file. Please add a level 1 heading to the top of the file and try again.`
```

Comment:
What do you think about this copy: `Expected file to start with a level 1 heading. A level 1 heading is important to for accessibility and layout consistency. Add a level 1 heading at the top of the document.`